CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `
 
 ### Changed
 
+- SC-6950 - changes validation for officialSchoolNumber
```

Comment:
validation for officialSchoonNumber now allows 4-6 numerical characters instead of 5.
?